public/video-ui/src/services/pandaReqwest.js

```diff
@@ -42,6 +42,15 @@ export function pandaReqwest(reqwestBody, timeout = 0) {
 
     // prettier-ignore
     if (payload.contentType === 'application/json' && typeof payload.data === 'object') {
+
+      //Replace all empty strings with nulls so that dynamo can save all the data we
+      //send to the server
+      Object.keys(payload.data).forEach(key => {
```

Comment:
I wasn't sure about this one, wouldn't want to have to do it in multiple components either. Maybe it should be in the VideoApi service so we could make sure that videos with empty strings for properties never get saved?